esphome/components/select/__init__.py

```diff
@@ -58,6 +62,30 @@
     }
 )
 
+_UNDEF = object()
+
+
+def select_schema(
+    class_: MockObjClass = _UNDEF,
+    *,
+    entity_category: str = _UNDEF,
+    icon: str = _UNDEF,
+):
+    schema = Schema({})
```

Comment:
All schema/voluptous apis should be accessed via cv/config_validation please.

```suggestion
    schema = cv.Schema({})
```

Reply:
thanks fixed. No idea where I got this code sample from actually